pkg/dmesg/commands.go

```diff
@@ -0,0 +1,146 @@
+package dmesg
+
+import (
+	"context"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/leptonai/gpud/pkg/process"
+)
+
+type Commands struct {
```

Comment:
Do components that use Commands need to call washCommands/tailcommands themselves?
Is it possible that hide these fields? Just let the component call the method watch dmesg data?
I'm curious what the difference is between Watch and Tail? Can I start watching from 1 hour ago? If yes, Tail can be deleted.